flight/Libraries/sanitycheck.c

```diff
@@ -76,23 +76,13 @@ int32_t configuration_check()
 	}
 
 	// Classify airframe type
-	bool multirotor = true;
```

Comment:
This is the kind of change I really like in the code. It makes it much cleaner to determine if something is a multirotor type, and will make it much easier to add new frame variants.